models/users.js

```diff
@@ -26,23 +26,11 @@ const UserSchema = new mongoose.Schema(
   }
 );
 
-/*
-  Note:
-    Only the functions of this model should be exported, not the actual model. All the database related methods should
-    only be in the models.
-
-  example: 
-  exports.query = function(pattern, sort, next) {
-    studentModel.find({ name: { $regex: pattern } }).sort(sort).exec( function(err, students){
-        next(students); // next() == *function callback at the controller*
-    });
-  };
-*/
 const UserModel = mongoose.model('users', UserSchema);
 
 //module.exports = mongoose.model('users', UserSchema); //this should not be exported
 
-exports.getUser = function (email, next) {
+exports.getUserByEmail = function (email, next) {
   UserModel.find({email: email}, (err, result) => {
     if (err) throw err;
     result.forEach((doc) => {
```

Comment:
The code implies that there's a possibility to get multiple users through this method.

Please consider using findOne()

Reply:
This is okay for now since every email on our current database would be unique. This is only used for the profile and editProfile. So for now, as is muna since if meron na logged in user, all the operations regarding the profile would be getUserByID.